Sandbox/imgui.ini

```diff
@@ -9,10 +9,19 @@ Size=92,48
 Collapsed=0
```

Comment:
ImGui preference files should not be tracked, in my opinion.

Reply:
Once we set up a docking viewports, we'll be commiting multiple ini files. This is just the default one for now. Rather have 1 'working' versus none at all